src/main/kotlin/snyk/amplitude/AmplitudeExperimentService.kt

```diff
@@ -19,11 +19,11 @@ private val LOG = logger<AmplitudeExperimentService>()
 class AmplitudeExperimentService : Disposable {
     companion object {
         const val CHANGE_AUTHENTICATE_BUTTON = "ide-jetbrains-change-authenticate-button-and-first-time-workflow"
-        const val TEST_GROUP = "TEST"
+        const val TEST_GROUP = "treatment"
```

Comment:
Why change it to `treatment` if we had it defined as `test` for the previous experiment? I'd rather rename the group on Amplitude side.

Also there's possibility that we will be having more groups when running A/B/C test in future.

Reply:
Treatment is the default on Amplitude - and I'd rather stick to their default. But in the end, I don't care much.